tfutils/plan_mapper.go

```diff
@@ -215,6 +235,20 @@ func MappedItemDiffsFromPlan(ctx context.Context, planJson []byte, fileName stri
 		results.Results = append(results.Results, mapResourceToQuery(itemDiff, currentResource, relevantMappings))
 	}
 
+	// Attach failed mappings to the span
+	for _, result := range results.Results {
+		switch result.Status {
+		case MapStatusUnsupported, MapStatusNotEnoughInfo:
+			span.AddEvent("UnmappedResource", trace.WithAttributes(
+				attribute.String("status", result.Status.String()),
+				attribute.String("message", result.Message),
+				attribute.String("terraform-name", result.TerraformName),
```

Comment:
```suggestion
				attribute.String("ovm.climap.status", result.Status.String()),
				attribute.String("ovm.climap.message", result.Message),
				attribute.String("ovm.climap.terraform-name", result.TerraformName),
```

always use the ovm prefix to make the attributes easier to locate in honeycomb